server/src/main/java/io/druid/server/coordinator/rules/IntervalLoadRule.java

```diff
@@ -41,9 +42,9 @@ public IntervalLoadRule(
       @JsonProperty("tieredReplicants") Map<String, Integer> tieredReplicants
   )
   {
-    validateTieredReplicants(tieredReplicants);
+    this.tieredReplicants = tieredReplicants == null ? ImmutableMap.<String, Integer>of() : tieredReplicants;
```

Comment:
I also think we should not allow empty replicants. It wouldn't make sense to have a load rule without specifying a place to put the data. `null` should mean "use the default" and empty should not be allowed.

The only caveat would be if someone created rules with empty tieredReplicant map in the past, that would cause the coordinator to fail miserably, as it did for us with null values.